src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyAmendmentService.cs

```diff
@@ -18,24 +18,47 @@ public class MethodologyAmendmentService : IMethodologyAmendmentService
         private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
         private readonly IUserService _userService;
         private readonly IMapper _mapper;
+        private readonly ContentDbContext _context;
 
         public MethodologyAmendmentService(
             IPersistenceHelper<ContentDbContext> persistenceHelper, 
             IUserService userService,
-            IMapper mapper)
+            IMapper mapper, 
+            ContentDbContext context)
         {
             _persistenceHelper = persistenceHelper;
             _userService = userService;
             _mapper = mapper;
+            _context = context;
         }
 
         public async Task<Either<ActionResult, MethodologySummaryViewModel>> CreateMethodologyAmendment(
             Guid originalMethodologyId)
         {
             return await _persistenceHelper.CheckEntityExists<Methodology>(originalMethodologyId)
                 .OnSuccess(_userService.CheckCanMakeAmendmentOfMethodology)
-                .OnSuccessDo(() => throw new NotImplementedException())
+                .OnSuccess(HydrateMethodologyForAmendment)
+                .OnSuccess(CreateAndSaveAmendment)
                 .OnSuccess(_mapper.Map<MethodologySummaryViewModel>);
         }
+        
+        // TODO EES-2156 - copy Methodology Files
```

Comment:
I can't see any kind of feature flag yet to disable creating amendments if you are still planning to do this copying as a separate change.

Reply:
It's in the front end - `MethodologySummary.tsx` has a flag to allow the button to be seen, which is currently only set to true in tests.

Reply:
Cool, thanks